Formula/mpi4py.rb

```diff
@@ -0,0 +1,30 @@
+class Mpi4py < Formula
+  desc "Python bindings for MPI"
+  homepage "https://mpi4py.readthedocs.io"
+  url "https://bitbucket.org/mpi4py/mpi4py/downloads/mpi4py-3.0.2.tar.gz"
+  sha256 "f8d629d1e3e3b7b89cb99d0e3bc5505e76cc42089829807950d5c56606ed48e0"
+  head "https://bitbucket.org/mpi4py/mpi4py.git", :branch => "master"
```

Comment:
Please get rid of the head block, unless you have a good reason for keeping it

Reply:
I was using head of mpi4py for several months until the 3.0.0 release was tagged, sometimes upstream MPI flavors have breaking changes that need adoption from nightly.
But can remove it, currently all is good.

Reply:
Ok, it sounds like maybe this would be ok to keep? 

If upstream (@dalcinl) added a nightly `brew install --HEAD mpi4py && brew test --HEAD mpi4py` test I’d definitely give it a thumbs up. 

But if you anticipate using/needing it then that could be ok.

Reply:
Okay, let's add this when needed I guess :)